packages/sirius-web/frontend/sirius-web-application/src/views/library-browser/EditLibraryView.tsx

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { SelectionContextProvider, Workbench } from '@eclipse-sirius/sirius-components-core';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { makeStyles } from 'tss-react/mui';
+import { EditProjectNavbar } from '../edit-project/EditProjectNavbar/EditProjectNavbar';
```

Comment:
Don't reuse things from the project support, those two don't need to be coupled together

Reply:
Done